source/Gesture Recognizer/Assets/Plugins/GestureRecognizer/Demo/Scripts/Menu.cs

```diff
@@ -0,0 +1,72 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+namespace GestureRecognizer.Demo
+{
+    public class Menu : MonoBehaviour
+    {
+        #region FIELDS
+
+        [SerializeField] private GameObject[] sections = null;
+        [SerializeField] private Button nextSectionButton = null;
+        [SerializeField] private Button previousSectionButton = null;
+
+        private int index = 0;
+
+        #endregion
+
+        #region PROPERTIES
+
+        private GameObject CurrentSection { get { return sections[index]; } }
+
+        #endregion
+
+        #region BEHAVIORS
+
+        private void Awake()
+        {
+            nextSectionButton.onClick.AddListener(NextSection);
+            previousSectionButton.onClick.AddListener(PreviousSection);
+            HideAllSections();
+            ShowCurrentSection();
+        }
+
+        private void NextSection()
+        {
+            HideCurrentSection();
+            index++;
+            if (index >= sections.Length)
```

Comment:
This looks like a good candidate for the `ternary` operator.

Reply:
Done